### ISSUE TITLE
Drag and drop options

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1674]
Relevant Components:

- [ ] Settings

Description
Many users cannot control the mouse well enough to drag and drop items accurately. Additionally, many users forego the use of a mouse at all. If there is drag and drop functionality on a page, it must be duplicated through other means, including for keyboard.

Provide alternate functionality for drag and drop sections.

Solution
Drag and drop cannot be the only way to operate a page. Provide another way, such as a “delete” button next to the item, which is accessible by keyboard.
